llvm-general/src/LLVM/General/Internal/FFI/TargetC.cpp

```diff
@@ -338,22 +316,4 @@ void LLVM_General_InitializeAllTargets() {
   // None of the other components are bound yet
 }
 
-LLVMBool LLVM_General_TargetMachineEmit(
```

Comment:
This function broke with the 3.9 update and iirc it also already broke with the 3.8 update. I don’t see any good reason to use this function directly and by limiting us to the API provided by the LLVM C API where possible, the amount of breakage to be expected will be smaller.